### ISSUE TITLE
Question about loss calculation

### ISSUE THREAD

**Issue by kevwang1** [codertimo/ConveRT-pytorch#25]
I notice you use `torch.nn.functional.cross_entropy` to calculate the loss. But doesn't that take the unnormalized class scores as input (scores before softmax)? To my understanding `torch.nn.functional.cross_entropy` will apply `log_softmax` internally. But in your loss calculation, you already apply softmax onto the output scores in `calculate_similarity`. Aren't you applying softmax twice then?

**Comment by codertimo** [comment]
@kevwang1 thanks for your feedback! I think you are right 👍 I fixed this bug on https://github.com/codertimo/ConveRT-pytorch/commit/f731887027b47bc2986c7199b8a07e42ab8800d0
https://github.com/codertimo/ConveRT-pytorch/blob/f731887027b47bc2986c7199b8a07e42ab8800d0/convert/model.py#L482-L529
